sdk/networkanalytics/arm-networkanalytics/test/networkanalytics_operations_test.spec.ts

```diff
@@ -60,51 +61,87 @@ describe("NetworkAnalytics test", () => {
 
   it("operation list test", async function () {
     const resArray = new Array();
-    for await (let item of client.operations.list()) {
+    for await (let item of client.operations.list().byPage()) {
       resArray.push(item);
     }
     assert.notEqual(resArray.length, 0);
   });
 
-  it.skip("dataProducts create test", async function () {
-    const res = await client.dataProducts.beginCreateAndWait(
+  it.only("dataProducts create test", async function () {
+    const res = await client.dataProducts.create(
+      subscriptionId,
       resourceGroup,
       resourcename,
       {
+        identity: {
+          type: "SystemAssigned"
+        },
         location,
         properties: {
           majorVersion: "1",
           product: "MCC",
           publisher: "Microsoft"
-        }
-      },
-      testPollingOptions);
```

Comment:
can we set this option in new code?

Reply:
it is not available for non-lro operation